Foundation/src/Environment_WIN32U.cpp

```diff
@@ -34,200 +33,206 @@
 namespace Poco {
 
 
-std::string EnvironmentImpl::getImpl(const std::string& name)
-{
-	std::wstring uname;
-	UnicodeConverter::toUTF16(name, uname);
-	DWORD len = GetEnvironmentVariableW(uname.c_str(), 0, 0);
-	if (len == 0) throw NotFoundException(name);
-	Buffer<wchar_t> buffer(len);
-	GetEnvironmentVariableW(uname.c_str(), buffer.begin(), len);
-	std::string result;
-	UnicodeConverter::toUTF8(buffer.begin(), len - 1, result);
-	return result;
-}
-
-
-bool EnvironmentImpl::hasImpl(const std::string& name)
-{
-	std::wstring uname;
-	UnicodeConverter::toUTF16(name, uname);
-	DWORD len = GetEnvironmentVariableW(uname.c_str(), 0, 0);
-	return len > 0;
-}
-
-
-void EnvironmentImpl::setImpl(const std::string& name, const std::string& value)
-{
-	std::wstring uname;
-	std::wstring uvalue;
-	UnicodeConverter::toUTF16(name, uname);
-	UnicodeConverter::toUTF16(value, uvalue);
-	if (SetEnvironmentVariableW(uname.c_str(), uvalue.c_str()) == 0)
+	std::string EnvironmentImpl::getImpl(const std::string& name)
```

Comment:
There is no need to indent all of the functions in this file. Please revert the indentation as it was before and as it is also used in other files.

Reply:
Huh, weird. I bet Visual Studio did this to me. Definitely didn't show it as a diff locally in VS, which is really annoying.

Reply:
Looks fixed now.